substrate/frame/core-fellowship/src/lib.rs

```diff
@@ -530,9 +532,59 @@ pub mod pallet {
 
 			Ok(Pays::No.into())
 		}
+
+		/// Set the parameters partially.
+		///
+		/// - `origin`: An origin complying with `ParamsOrigin` or root.
+		/// - `partial_params`: The new parameters for the pallet.
+		///
+		/// This update config with multiple arguments without duplicating
+		/// the fields that does not need to update (set to None).
+		#[pallet::weight(T::WeightInfo::set_params())]
```

Comment:
To be precise it should have its own benchmark.

Reply:
Resolved in https://github.com/paritytech/polkadot-sdk/pull/3843/commits/e7c33076bbadca00420b66cd32ba4fc4951672c4